src/Runner.jl

```diff
@@ -918,12 +918,12 @@ function platform_envs(platform::AbstractPlatform, src_name::AbstractString;
         ), ":"),
 
         "LD_LIBRARY_PATH" => join((
-           # Start with a default path
-           "/usr/local/lib64:/usr/local/lib:/usr/lib64:/usr/lib",
+            # Start with our CSL libraries for all architectures that can natively run within this environment
+            csl_paths(host_platform),
+            # Then add default system paths
+            "/usr/local/lib64:/usr/local/lib:/usr/lib64:/usr/lib",
```

Comment:
Honestly, I'm not 100% this is a good idea.  Currently we have:
```console
sandbox:${WORKSPACE} # /usr/bin/gcc --version
gcc (Alpine 9.3.0) 9.3.0
Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```
Maybe we should move our CSL forward only if our host GCCBootstrap shard is > 9.3?

Reply:
Hmmmm.  So the struggle here is that we want to pick up e.g. `libgcc_s` that matches the host compiler?

We shouldn't really be using `/usr/bin/gcc` at all, right?  Unless we're bootstrapping....

Reply:
No, but the problem is that in https://github.com/JuliaPackaging/Yggdrasil/pull/3221#issuecomment-869701371 system libstdc++.so was picked, when we were building with our GCC 11, so that running an application would result in the now infamous undefined symbol `_ZSt28__throw_bad_array_new_lengthv@@GLIBCXX_3.4.29`